src/tracebox/tracebox.cc

```diff
@@ -54,103 +53,91 @@ const Applet g_applets[]{
     {"perfetto", PerfettoCmdMain},
     {"trigger_perfetto", TriggerPerfettoMain},
     {"websocket_bridge", WebsocketBridgeMain},
+    {"ctl", TraceboxCtlMain},
 };
 
 void PrintUsage() {
   printf(R"(Welcome to Perfetto tracing!
 
 Tracebox is a bundle containing all the tracing services and the perfetto
-cmdline client in one binary. It can be used either to spawn manually the
-various subprocess or in "autostart" mode, which will take care of starting
-and tearing down the services for you.
+cmdline client in one binary.
 
-Usage in autostart mode:
-  tracebox -t 10s -o trace_file.perfetto-trace sched/sched_switch
-  See tracebox --help for more options.
+QUICK START (end-to-end workflow):)");
 
-Usage in manual mode:
-  tracebox applet_name [args ...]  (e.g. ./tracebox traced --help)
-  Applets:)");
+#if !PERFETTO_BUILDFLAG(PERFETTO_OS_WIN)
+  printf(R"(
+  tracebox ctl start                      # Start daemons
+  tracebox -t 10s -o trace.pftrace sched  # Capture trace
+  tracebox ctl stop                       # Stop daemons (optional)
+)");
+  PrintTraceboxCtlUsage();
+  printf(R"(
+ADVANCED: --autodaemonize (NOT RECOMMENDED)
```

Comment:
I don't want to call this "advanced" as I think it's legit people want this if they just are looking at ftrace. I would present them as two equal options but with preference to daemonization.